backend/main/shared/snapshot-flow.go

```diff
@@ -23,13 +23,35 @@ type Snapshot struct {
 	Finished     time.Time `json:"finished"`
 }
 
+// copied this quick and dirty from models to get around circular dependency
+// move to shared?
+type Balance struct {
```

Comment:
Yeah probably cleanest to move this to `shared/structs.go` and use it in `balance.go` and `snapshot-flow.go` if that works